### DISCUSSION TITLE
aiohttp send_bytes or send_str hangs out

### DISCUSSION THREAD

**Discussion by redradist** [aio-libs/aiohttp#7562] [Q&A]
It happens that sometime `send_bytes` or `send_str` (`aiohttp` version 3.8.3) hangs for long time ...
Unfortunately, do not have `timeout` argument in `send_bytes` or `send_str` as we have in `receive` function

Have anybody faced the same issue ?
How to workaround it ??

**Comment by redradist** [comment]
I have found these latest changes in 3.8.4 https://github.com/aio-libs/aiohttp/pull/7180
Maybe it is fixing the issue ...

**Reply by Dreamsorcerer** [reply]
Well, did it fix the issue, or are you still experiencing it...?

Some kind of reproducer or more information is definitely needed.
